src/main/java/net/landofrails/landofsignals/LandOfSignals.java

```diff
@@ -28,7 +28,7 @@ public class LandOfSignals extends ModCore.Mod {
     @SuppressWarnings({"java:S1845"})
     public static final String MODID = "landofsignals";
     // Current version
-    public static final String VERSION = "1.0.2";
+    public static final String VERSION = "1.0.3";
```

Comment:
There is no need to change the version here anymore. It's handled by the release pipeline